src/components/charts/ChartGroupUpdate.tsx

```diff
@@ -210,7 +210,7 @@ export default function ChartGroupUpdate({ }) {
                             <div style={{ height: "calc(100vh - 150px" }}>
                                 <EmptyState>
                                     <EmptyState.Image><img src={emptyImage} alt="" /></EmptyState.Image>
-                                    <EmptyState.Title><h4>No  matching Charts</h4></EmptyState.Title>
+                                    <EmptyState.Title><h4>No matching Charts</h4></EmptyState.Title>
```

Comment:
I think `Charts` should be in a small case. Please confirm & change.

Reply:
done